cmd/tether/attach.go

```diff
@@ -179,131 +168,147 @@ func (t *attachServerSSH) stop() {
 func (t *attachServerSSH) run() error {
 	defer trace.End(trace.Begin("main attach server loop"))
 
-	var sConn *ssh.ServerConn
+	var serverConn *ssh.ServerConn
 	var chans <-chan ssh.NewChannel
 	var reqs <-chan *ssh.Request
 	var err error
 
-	// keep waiting for the connection to establish
-	for t.enabled && sConn == nil {
-		conn := t.conn
-		if conn == nil {
-			// Stop has probably been called as t.conn is set in Start and should
-			// never be nil otherwise
-			err := fmt.Errorf("connection provided for backchannel is nil")
-			log.Debug(err.Error())
-			return err
-		}
+	for t.enabled {
+		// keep waiting for the connection to establish
+		for serverConn == nil {
```

Comment:
This is not new - I mention it because I'm not sure what happens when the port is disconnected. I *think* we should just block as normal, but ....